tests/sqlparser_hive.rs

```diff
@@ -32,6 +32,20 @@ fn parse_table_create() {
     hive().verified_stmt(iof);
 }
 
+fn generic(options: Option<ParserOptions>) -> TestedDialects {
+    TestedDialects {
+        dialects: vec![Box::new(GenericDialect {})],
```

Comment:
Can you also please  test `HiveDialect` given this is in the `sqlparser_hive.rs` module?

Reply:
I've looked it over and colocated the dialects. I didn't find any test-cases that tested `generic` and not `hive`. Everything was either `hive` alone or `hive` and `generic`, e.g. the `parse_create_function` test. I'm not sure it makes sense to blend them given how they get used in the tests. I think it makes sense to test hive-isms that `generic` should be able to handle in the module that tests hive-isms rather than duplicating everything into the generic test module.

Reply:
I don't mind doing a follow-up PR.

Perhaps I could do something like `parse_deeply_nested_expr_hits_recursion_limits` in the common module but parameterized over the dialect, then invoke the helper in the generic & hive test modules? If that sounds acceptable let me know and I will pursue that.

Reply:
What I was suggesting was to remove `generic()` and test `parse_describe` with both hive and generic